pkg/server/commands/list_objects.go

```diff
@@ -296,7 +307,7 @@ func trySendObject(object string, objectsFound *uint32, maxResults uint32, resul
 func (q *ListObjectsQuery) Execute(
 	ctx context.Context,
 	req *openfgav1.ListObjectsRequest,
-) (*openfgav1.ListObjectsResponse, error) {
+) (*ListObjectsResponse, error) {
```

Comment:
Why don't we leave this as is and move this down into the ConnectedObjects layer? Our pattern has always been to expose the same signature for the RPC in the commands package. For consistency it seems we should avoid changing that pattern.

A `ConnectedObjectsResult` should have metadata associated with the resolution info, and using that we can calculate the overall query count by taking the sum of all of the query counts from each result.

Reply:
For Check, we were able to "get away" with ResolveCheck response being different than the RPC signature name because the check code lives inside the server module.  However, list object has both server and commands package code. 

As noted below, we could not use  ConnectedObjectsResult because the resolution metadata needs to per request not per ListObject response.  Otherwise, it may overcount in the case of multiple objects or undercount if it does not have any object in the response.

Reply:
> Why don't we leave this as is and move this down into the ConnectedObjects layer?

We could do this, but then the logging of the resolution metadata can't happen in the `server.go` file, it must happen within the `ListObjects` code (which i'm okay with, for me, the less code we have in `server.go`, the better)

Reply:
Not only that, we will also need to move the trace inside the ListObjects code as well.  However, as I noted above, the biggest problem is the resolution metadata needs to be on a per request level, not on a per connected objects result level.  Otherwise, it is subject to overcount/undercount.